src/main/groovy/com/jfrog/bintray/gradle/BintrayUploadTask.groovy

```diff
@@ -173,10 +173,7 @@ class BintrayUploadTask extends DefaultTask {
     @TaskAction
     void bintrayUpload() {
         logger.info("Gradle Bintray Plugin version: $pluginVersion");
-        if (shouldSkip()) {
-            logger.info("Skipping task '{}:bintrayUpload' because user or apiKey is null.", this.project.name);
-            return
-        }
+        assertUploadConfiguration()
```

Comment:
**Warning** to bintray plugin owners: this change is backwards incompatible. This change makes sense from the standpoint of fast feedback and the principle of least surprise (silently skipping is bad). If you choose to roll out incompatible change please document it well and apply semver principles (bump major version).

Reply:
If JFrog does not want to make a missing API key or user fail the build, then at least log a warning instead of silently failing. Just changing `logger.info` to `logger.warn` would save a lot of people a lot of time as warnings would be logged by Gradle by default.